package/Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.400
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.100+up0.1.1
```

Comment:
to match the latest version in https://github.com/rancher/charts/tree/dev-v2.5/charts/rancher-webhook/rancher-webhook